docs/index.md

```diff
@@ -4,6 +4,8 @@ hide:
 - toc
 ---
 
+<!-- markdownlint-disable MD033 MD041 -->
```

Comment:
The page doesn't have an h1 title and uses HTML tags.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md033.md
https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md041.md